remote-config.jsonc

```diff
@@ -251,6 +242,42 @@
         },
         {
           "message": "Don't forget to upgrade DDEV! https://ddev.readthedocs.io/en/stable/users/install/ddev-upgrade/"
+        },
+        {
+          "message": "DDEV goes to great lengths to keep you and your projects working. See the recent event where MariaDB changed their dump file format, https://ddev.com/blog/mariadb-dump-breaking-change"
+        },
+        {
+          "message": "`ddev .` is a shortcut for `ddev exec`, so you can `ddev . ls` for example."
```

Comment:
is that tip necessary at all? there is already line 54?

Reply:
Let's mention `ddev aliases` instead, so users know what they can use.

Reply:
I added it.